bin/cron/deliver_poste_messages

```diff
@@ -342,14 +343,22 @@ def main
   workers.each(&:join)
 
   remaining_count = POSTE_DB[:poste_deliveries].where(sent_at: nil).count
+  abandon_count = POSTE_DB[:poste_deliveries].where(sent_at: 0).count
+  sent_count = POSTE_DB[:poste_deliveries].where(Sequel.lit('sent_at >= ?', started_at)).count
+
+  if CDO.newrelic_logging
+    # How many emails we sent on _this run_ of the cronjob
+    NewRelic::Agent.record_metric("Custom/Poste/Sent", sent_count)
+    # How many total abandoned emails we have
+    NewRelic::Agent.record_metric("Custom/Poste/Abandoned", abandon_count)
+    # How many emails are still queued for send on subsequent runs
+    NewRelic::Agent.record_metric("Custom/Poste/Queued", remaining_count)
+  end
 
-  elapsed_seconds = ((Time.now - started_at) * 100.0).to_i / 100.0
-
-  unless queue_count == 0 && remaining_count == 0
+  unless queue_count == 0 && remaining_count == 0 && rack_env?(:development)
```

Comment:
just curious: what is the reason the code has always suppressed the message only when the counts are zero and we're on the development environment?

Reply:
oh actually, didn't we previously only `puts` on development?

Reply:
This is equivalent to `if queue_count != 0 || remaining_count != 0 || rack_env?(:development)` then `puts` isn't it?
(For some reasons every timeI see `unless` at the beginning of a block of code I have to translate it into an `if` block.)

Reply:
:man_facepalming: I definitely mixed up the `if` and `unless` cases here.  I'd actually love to just tear out this development-only code anyway - thoughts on that?

Reply:
The message seems useful for manual runs of this cron job, but not for any other case.  I wouldn't mind it being removed.